tests/handlers/test_presence.py

```diff
@@ -909,8 +909,14 @@ def test_send_and_get(self) -> None:
 
         prev_token = self.queue.get_current_token(self.instance_name)
 
-        self.queue.send_presence_to_destinations((state1, state2), ("dest1", "dest2"))
```

Comment:
Here and below: was the test broken by not having `get_success` (or similar) here?

Reply:
Yeah, as calling a coroutine doesn't do anything unless you await on it